clients/fides-js/src/lib/initialize.ts

```diff
@@ -463,29 +463,31 @@ export const initialize = async ({
          * don't block the rest of the code from executing, we use setTimeout with
          * no delay which simply moves it to the end of the JavaScript event queue.
          */
-        setTimeout(() => {
-          automaticallyApplyGPCPreferences({
-            savedConsent,
-            effectiveExperience: effectiveExperience as PrivacyExperience,
-            cookie,
+        setTimeout(
+          automaticallyApplyGPCPreferences.bind(null, {
```

Comment:
This method has the potential to alter whether or not a banner/modal will show in the UI. 

E.g. a new user visits the site with GPC enabled in their browser. The only notice that Fides returns in the experience is GPC enabled. First, Fides calls `FidesInitialized` twice, where `shouldShowExperience` is true. Then Fides hits this line and automatically saves this preference in the API and cookie, and does not surface the banner/modal. 

So, in this case, Fides will send the `FidesUpdated` event automatically, without the user explicitly selecting a consent option. We _could_ pass in `shouldShowExperience` in this event, too, but it really depends on how this var will be queried by the customer

Reply:
OK so I just confirmed with @NevilleS that GPC shouldn't affect the customer they are specifically using TCF, but this is something we'll want to update in the future. So the approach you have here is fine for now 👍

Reply:
Interesting.. Thanks for explaining this! That's another situation where `shouldShowExperience` is not going to match the actual behavior. Another one that I'm aware of is setting `fides_disable_banner` to true. This is not an exact property. Perhaps in the future we could just run the view logic (i.e., just attempt to render the banner and use that to determine the value of `shouldShowExperience`).